client-src/elements/chromedash-userlist.ts

```diff
@@ -75,14 +71,17 @@ class ChromedashUserlist extends LitElement {
   }
 
   _onAdminToggle() {
-    const formEl = this.shadowRoot.querySelector('form');
-    const adminCheckbox = formEl.querySelector('input[name="is_admin"]');
-    const siteEditorCheckbox = formEl.querySelector(
+    const formEl: HTMLFormElement | null =
```

Comment:
I believe you don't need the explicit type here. `querySelector` has a smart enough type to get it right:
```suggestion
    const formEl =
```